README.md

```diff
@@ -136,267 +132,43 @@ For an even more in-depth demo, check out our [Deploying to Kubernetes](https://
+
+### GitHub releases
+Post-1.0, GitHub releases are created for GitHub tagged versions that have undergone additional quality activities to ensure that Secretless continues to meet its [stability requirements](#stable-release-definition). When a GitHub release is created for a stable Secretless version, an additional Docker image is pushed to DockerHub with the `stable` tag.
+
+### GitHub repository
+The code on `master` in the project's GitHub repository represents the development work done since the previous GitHub tag. It is possible to build Secretless from source (see [our contributing guidelines](#development) for more info), but for regular use we recommend using the `stable` Docker image from DockerHub or an official GitHub release.
+
+### Stable release definition
+Stable components (eg service connectors, credential providers, etc.) of Secretless meet the core acceptance criteria:
+- Synchronous user actions must complete in a reasonable amount of time
+- The component must guard against threats from all parts of the [STRIDE threat model](https://en.wikipedia.org/wiki/STRIDE_(security))
+- Documentation exists that clearly explains how to set up and use the component as well as providing troubleshooting information for anticipated common failure cases.
+- A suite of automated tests that exercise the component exists, and code coverage has been measured. The entire team is in mutual agreement that any code not tested by the unit test suite is acceptable to leave untested. Tests that are deferred to be implemented at a later time are tracked so that we don't lose visibility into that work.
+  
+In addition, the following must be true for any stable release:
+- Secretless has had security review, and has no high or critical bugs
+  - Secretless has been scanned for CVEs and security vulnerabilities with no high issues
+  - Secretless has undergone threat modeling
+-	Secretless has a stable API and configuration spec and there will be no near-term backwards-incompatible changes
+- For use cases involving stable components of Secretless (eg service connectors, credential providers, etc.):
+  -	Secretless is stable while running
+    -	It does not randomly drop connections while running
+    -	It can run without failure for an extended period of time (~4 days)
+    -	It can consistently open connections while running
+    -	When it fails in Kubernetes / OpenShift, the pod is destroyed and rescheduled
+  -	Secretless has minimal impact on connection speeds
+    -	Request throughput is within 20% of direct-to-DB speed
+  -	Secretless performs under load in a realistic environment
+  -	Secretless handles connections transparently, so that requests from the client are appropriately transmitted to the server and messages from the server are propagated back to the client
+-	Secretless is easy to set up, and setup takes less than 30 minutes
```

Comment:
This needs a broader definition on what that setup is

Reply:
"and [adding the configured secretless broker sidecar to your application]"?

Reply:
LGTM!